<launch>
    <node name="joy_node" pkg="joy" type="joy_node" >
        <param name="dev" type="string" value="/dev/input/js0"/>
    </node>

    <node name="Xbox_controller_Node" pkg="xbox_controller" type="Xbox_controller_Node" output="screen">
    </node>    
</launch>
